web/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@openshift-console/dynamic-plugin-sdk": "^0.0.18",
+    "@openshift/dynamic-plugin-sdk-webpack": "^2.0.0",
```

Comment:
Just a nit pick, this should be a dev dependency.

Reply:
fixed